docs/src/main/sphinx/release/release-469.md

```diff
@@ -0,0 +1,110 @@
+# Release 469 (8 Jan 2025)
+
+## General
+
+* Refrain from join pushdown for modified tables ({issue}`24447`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Improve performance of reading from Delta Lake tables by compressing files
```

Comment:
@raunaqmorarka .. how does this work with existing stuff. I assume the files are unmodified as usual so it only applies to new data .. so reading is NOT accelerated for old data .. only for newly written files that are subsequently read .. and as such.. do we need to reword @martint .. also note I added the info about SNAPPY being the old default.